### ISSUE TITLE
low: from method

### ISSUE THREAD

**Issue by betula** [betula/realar#39]
Add synthetic value creator function

```javascript

const v = synthetic(set, get); // make new synthetic value

v.val
v.update
v.reset !! //no defined here)

```

Or provide possibility for creation both synthetic entities value and signal.

```javascript

const s = signal.synthetic(set, get);
// or
const s = signal.from(set, get); // without reset method

// and of course

const v = value.synthetic(set, get);
// or
const v = value.from(set, get);

const r = ready.synthetic(set, get);
// or
const r = ready.from(set, get);

```

**Comment by betula** [comment]
```javascript

(signal|value|ready|...).from(
  signal|value|ready
);

(signal|value|ready|...).from(
  set, get
);

```

**Comment by betula** [comment]
```javascript

value.from(get) - readonly value
signal.from(get) - readonly signal

// possible to use other reactive values as get or set methods.

value.from(write_value, read_value);

// Think about the order

value.from(get, set);
// or
value.from(set, get);

// But because writeonly values are no possible, one argument must be only get function

value.from(get) - readonly value

```

**Comment by betula** [comment]
`ready.from` added https://github.com/betula/realar/commit/392a6a227c0f7130d67e10a8536475400093c394

**Comment by betula** [comment]
Should be added:
`ready.flag.from`

**Comment by betula** [comment]
"signal.from" and "value.from" implemented in 0.6.0

The proposal for "ready.flag.from" added to "the stream of conciseness 0.7+ roadmap"
